src/Compilers/Test/Utilities/CSharp/CompilationTestUtils.cs

```diff
@@ -331,9 +332,12 @@ internal static void VerifyTypes(this CSharpCompilation compilation, SyntaxTree
 
                 string toDisplayString(SyntaxNode syntaxOpt)
                 {
-                    return (syntaxOpt != null) && dictionary.TryGetValue(syntaxOpt, out var type) ?
-                        (type.IsNull ? "<null>" : type.ToDisplayString(TypeSymbolWithAnnotations.TestDisplayFormat)) :
-                        null;
+                    // We don't support VerifyTypes on suppressions at the moment
+                    Debug.Assert(syntaxOpt.Kind() != SyntaxKind.SuppressNullableWarningExpression);
```

Comment:
`Assert.NotEqual(syntaxOpt.Kind(), SyntaxKind.SuppressNullableWarningExpression);`

Reply:
This was intentional. It's documenting the contract of the local function, rather than a test assertion. But I don't mind either way...

---
In reply to: [252035524](https://github.com/dotnet/roslyn/pull/32850#discussion_r252035524) [](ancestors = 252035524)